client/packages/invoices/src/Returns/api/operations.graphql

```diff
@@ -120,6 +120,16 @@ query invoiceByNumber($invoiceNumber: Int!, $storeId: String!) {
       lines {
         nodes {
           id
+          returnReasonId
```

Comment:
hmm, just a side note, i was looking at other invoiceByNumber queries, and they look pretty identical, would be great to perhaps consolidate in the future, also they are called like outboundByNumber or inboundByNumber, but now we have outboundReturn and outboundShipment, and this one is called invoiceByNumber (i think good to rename to outboundReturnByNumber at some point and potentially outboundShipment by number)

Reply:
I agree. I think consistent naming would allow us to more readily identify possible duplication.

Reply:
Ah yeah great point, perhaps can rename as part of the frontend work with the detail view. I think a naming review in a separate PR at the end might be a good idea, I think there are a couple of inconsistencies like that!